src/mots/cli.py

```diff
@@ -302,4 +251,35 @@ def create_parser():
     )
     export_parser.set_defaults(func=export)
 
+    # Add path argument for all parsers that need it.
```

Comment:
This works, but I would probably write a function like `add_path_argument` and then call it on each parser, so each parser is defined in the same block.